### DISCUSSION TITLE
RG/FD Config(s)

### DISCUSSION THREAD

**Discussion by barrettruth** [ibhagwan/fzf-lua#1288] [General]
I have an RG config as follows in `~/.config/rg/config`. From CLI, rg respects it but `fzf-lua` does not.

Similarly with FD and `~/.config/fd/config.toml` with `export FD_CONFIG_PATH=~.config/fd/config.toml`.

Wondering your input on these. Can attach images/cases if needed - but the main point is any rules set in these config files are ignored.

**Comment by ibhagwan** [comment]
That’s due to https://github.com/ibhagwan/fzf-lua/commit/f438477d3628fcf7a8959dfdb96f0564193cf4d9  post a decision made in #1266, the pros seem outweigh the cons, as opposed to `FZF_DEFAULT_OPTS`, we couldn’t find any reason to use the config file that couldn’t be solved with a simple addition to `grep.rg_opts`, the cons on the other hand are many, this can affect the line formatting eventually breaking fzf-lua.

**Reply by ibhagwan** [reply]
With https://github.com/ibhagwan/fzf-lua/commit/f5240e364df3d3b954205aecf0c860cd7b185c06 you can :
```lua
require("fzf-lua").setup({
  grep = { RIPGREP_CONFIG_PATH = vim.env.RIPGREP_CONFIG_PATH }
})
```

The commit also makes the prepration for `FD_CONFIG_PATH` though it does seem to not be taken into account AFAIK?
https://github.com/sharkdp/fd/issues/362

**Reply by dotfrag** [reply]
I'm not aware of an fd config file either. I think the only file it reads is the ignore file.
